Rakefile

```diff
@@ -28,6 +28,70 @@ task :release do
   sh "git release"
 end
 
+require "securerandom"
+require "./lib/cipherstash/index"
+require "json"
+
+task :generate_ordered_string_test_cases do
+  id = SecureRandom.uuid
+  settings = {
+    "meta" => {
+      "$indexId" => id,
+      "$indexName" => "titleSort",
+      "$prfKey" => SecureRandom.hex(16),
+      "$prpKey" => SecureRandom.hex(16),
+    },
+    "mapping" => {
+      "kind" => "range",
+      "field" => "title",
+      "fieldType"=>"string",
+    }
+  }
+  schema_versions = {:first=>0, :last=>0, :searchable=>true}
+
+  index = CipherStash::Index.generate(id, settings, schema_versions)
+
+  random_ascii_string = -> () do
```

Comment:
I presume this is a proc, rather than a "real" method, because you're trying to keep related things "in-scope"?  Given that this is a non-trivial chunk of code, it's probably better just to pull all this logic out into a separate class and use an instance of it in the Rakefile, to avoid having all this code directly in the Rakefile making the place look untidy.